zeroex/orderwatch/order_watcher.go

```diff
@@ -506,7 +507,7 @@ func (w *Watcher) generateOrderEventsIfChanged(hashToOrderWithTxHashes map[commo
 	orderEvents := []*zeroex.OrderEvent{}
 	for _, acceptedOrderInfo := range validationResults.Accepted {
 		orderWithTxHashes := hashToOrderWithTxHashes[acceptedOrderInfo.OrderHash]
-		txHashes := make([]common.Hash, len(orderWithTxHashes.TxHashes))
+		txHashes := []common.Hash{}
 		for txHash := range orderWithTxHashes.TxHashes {
```

Comment:
I'd still like to follow the convention of pre-allocating slice lengths when possible. What do you think of this solution?

```
txHashes := make([]common.Hash, len(orderWithTxHashes.TxHashes))
txHashIndex := 0
for txHash := range orderWithTxHashes.TxHashes {
    txHashes[txHashIndex] = txHash
    txHashIndex += 1
}
```

Reply:
It's more verbose but I guess I'm fine with it.